template/component.h.erb

```diff
@@ -65,6 +65,8 @@ struct common_data_s {
   /* Handling the downstream message queue */
   struct el *queue;
   struct el *pool;
+  /* Guarantee btx_usr_finalize in Filter is called just once. */
+  bt_bool finalized;
```

Comment:
Please add a comment when we are allocationing the struct to say that because we use calloc it will be `false`, or just do it explicitly just to be clear.

Reply:
Can we initialize the value to BT_FALSE in filter_initialize ?

Reply:
Yes exactly.